staging/cse/windows/containerdfunc.ps1

```diff
@@ -158,7 +158,7 @@ function Install-Containerd {
   $formatedbin = $(($CNIBinDir).Replace("\", "/"))
   $formatedconf = $(($CNIConfDir).Replace("\", "/"))
   $sandboxIsolation = 0
-  $windowsVersion = Get-WindowsVersion
+  $windowsVersion = Get-WindowsPauseVersion

```

Comment:
FYI: It will not work until you update cse script package version.

Reply:
I see. I will build a new cse script package after the code is merged.

Reply:
Should I move the change that returns "23H2" for 23H2 in Get-WindowsVersion to the pr of updating cse script package to avoid blocking issue?